sale_tax_round_down/tests/test_sale_tax_round_down.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestSaleTaxRoundDown(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestSaleTaxRoundDown, cls).setUpClass()
+        cls.product = cls.env["product.product"].create({"name": "Test Product"})
+        cls.partner = cls.env.ref("base.res_partner_1")
+        cls.tax = cls.env["account.tax"].create(
+            {
+                "name": "Tax 10",
+                "type_tax_use": "sale",
+                "amount": 10,
+            }
+        )
+        cls.company_jpy = cls.env["res.company"].create(
+            {
+                "name": "Japan company",
+                "currency_id": cls.env.ref("base.JPY").id,
+                "country_id": cls.env.ref("base.jp").id,
+            }
+        )
+
+    def test_sale_tax_round_down_with_true(self):
```

Comment:
```suggestion
    def test_sale_tax_round_down_with_true(self):
        assert.Equal(self.company_jpy.need_tax_round_down, True)
```
To make the assuption clear in the scenario.